README.md

```diff
@@ -169,11 +169,23 @@ assertHint(R.id.edittext, "Hint");
 // Check if text on screen contains given text
 assertContains("text");
 assertContains(R.id.textview, "text");
- 
+
 // ...or not?
 assertNotContains("text");
 assertNotContains(R.id.textview, "text");
 
+// Check text is given color
+assertTextColorIs(R.id.textViewRed, R.color.textRed); // color
+assertTextColorIs(R.id.textViewBlue, R.color.textBlue); // color
+assertTextColorIs(R.id.textViewStateRed, R.color.text_state_red); // color state list
+assertTextColorIs(R.id.textViewStateBlue, R.color.text_state_blue); // color state list
```

Comment:
Are these duplicated? If the signature is the same, I think one single sample is enough.

Reply:
Uooops